schema/changelog-4.0.xml

```diff
@@ -94,8 +112,69 @@
         <where>web = 0 AND mail = 0 AND sms = 1</where>
     </update>
 
+  </changeSet>
+
+  <changeSet author="author" id="changelog-4.0-pg">
+
+    <preConditions onFail="MARK_RAN">
+      <dbms type="postgresql" />
+      <columnExists tableName="tc_notifications" columnName="web" />
+      <columnExists tableName="tc_notifications" columnName="mail" />
+      <columnExists tableName="tc_notifications" columnName="sms" />
+    </preConditions>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="web,mail,sms" />
+        <where>web = TRUE AND mail = TRUE AND sms = TRUE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="web,mail" />
+        <where>web = TRUE AND mail = TRUE AND sms = FALSE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="web" />
+        <where>web = TRUE AND mail = FALSE AND sms = FALSE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="web,sms" />
+        <where>web = TRUE AND mail = FALSE AND sms = TRUE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="mail,sms" />
+        <where>web = FALSE AND mail = TRUE AND sms = TRUE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="mail" />
+        <where>web = FALSE AND mail = TRUE AND sms = FALSE</where>
+    </update>
+
+    <update tableName="tc_notifications">
+        <column name="notificators" value="sms" />
+        <where>web = FALSE AND mail = FALSE AND sms = TRUE</where>
+    </update>
+
+  </changeSet>
+
+  <changeSet author="author" id="changelog-4.0-post">
```

Comment:
Can we just name this one `changelog-4.0`? So that we don't have one extra record?

Reply:
It would also mean that we don't need a check.